modules/serverless-hotfix-patch.adoc

```diff
@@ -0,0 +1,166 @@
++
+After a new operator release is available, deployed operators with hotfixes can also be upgraded. To use the latest GA version, modify the subscription to use the public CatalogSource instead of the hotfix one.
+
+The following diagram illustrates how patches and hotfixes work:
+
+[source]
+----
+                                                                               
+     stable                                       stable-1.28                  
++--------------+                 +--------------------------------------------+
+|              |                 |                                            |
+|  +--------+  | corresponds to  |   +--------+    +--------+    +--------+   |
+|  | 1.28.0 |----------------------> | 1.28.0 |    | 1.28.1 |    | 1.28.2 |   |
+|  +--------+  |                 |   +--------+    +--------+    +--------+   |
+|              |                 |     |                   ^            |     |
+|              |                 +-----|-------------------|------------|-----+
+|  +--------+  |                created|                   |upgrades    |      
+|  | 1.28.1 |  |                from   |     hotfix_xyz    |to          |      
+|  +--------+  |                       |   +------------+  |            |      
+|              |                       +-->|            |--+            |      
+|              |                           |            |               |      
+|  +--------+  | upgrades to               +------------+               |      
+|  | 1.29.0 |<----------------------------------------------------------+      
+|  +--------+  |                                                               
+|              |                                                               
+|              |                                                               
+|  +--------+  |                                                               
+|  | 1.30.0 |  |                                                               
+|  +--------+  |                                                               
+|              |                                                               
++--------------+                                                               
+----
+
+== Upgrade path for maintenance releases
+
+If you use a version-based channel, you can always upgrade to the latest version in the channel, or head, for example, from 1.29.0 to 1.29.2 in the `stable-1.29` channel.
+
+Additionally, from the head of the channel, you can upgrade to the next `x.y` release. For example, if 1.29.2 is the head in the `stable-1.29` channel, you can upgrade from 1.29.2 to 1.30. Such cross-channel updates are not done automatically, and the administrator needs to switch the channel manually by updating the subscription.
+
+== Upgrade examples
+
+=== Scenario 1
+
+In this scenario:
```

Comment:
The [ISG](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#lead-in-wording) recommends we avoid list introductions that do not form a complete sentence: "To avoid possible translation problems, use complete sentences to introduce lists and not phrases". Here's one way to work around that:

```suggestion
In this scenario, the following circumstances are true:
```